src/components/status/AvailabilitySingleRegionPage.js

```diff
@@ -30,6 +30,32 @@ export default class AvailabilitySingleRegionPage extends Component {
     });
   }
 
+  render() {
```

Comment:
Can we use more React way here: 

- Create new parametrized element (that accept what resource to look) 
- Wrap with it all elements 

That should reduce amount of code.. .